lesson04/assignment/animationSources.md

```diff
@@ -0,0 +1,10 @@
+Animation & Transition Sources
+
+• Button transition "Shiny Effect"
+https://cssanimation.rocks/transition-vs-animation/
+
+• Preloader
+https://codepen.io/maximakymenko/pen/mdbzNMv
+
+• Rain
+https://codepen.io/arickle/pen/XKjMZY
```

Comment:
Very nice documentation of sources! This would also make sense to include in the README of a normal project (one where the README isn't full of instructions).

Reply:
Good to know, thanks Cheri!